### DISCUSSION TITLE
ipxe-devel Archives not longer updated?

### DISCUSSION THREAD

**Discussion by berlin4apk** [ipxe/ipxe#445] [Q&A] [answered]
Hey is the ipxe-devel Archives https://lists.ipxe.org/pipermail/ipxe-devel/
Not longer updated, as they are no archive for August 2021 or it's just that no mail was send this month?

**Comment by NiKiZe** [comment, marked as answer]
There have not been any emails to the devel list yet in august, Most activity has been on Github.

**Reply by berlin4apk** [reply]
Ok thanks for the Info
